### ISSUE TITLE
Include graphs

### ISSUE THREAD

**Issue by hieronymousch** [jooola/zabbix-matrix-webhook#6]
Hi,

thanks for your work. Do you see the integration of graphs as a possibility? 
thanks

**Comment by jooola** [comment]
Hey, my pleasure.

Right now, I don't know if it is doable without too much dirty hacking. The ideal solution would be for zabbix to provide a graph within the web-hook context.

See https://support.zabbix.com/browse/ZBXNEXT-5611 and https://support.zabbix.com/browse/ZBXNEXT-562

I don't plan to add such complexe hacking code in the webhook, as I would like to keep it simple. Maybe such feature could land in a dedicated alert script, as it allow more advanced logic than a webhook.

Maybe searching for how other web-hooks implemented this feature could give a idea on how to do this.

TLDR; this feature needs more info; yes, if Zabbix makes it simple to integrate; maybe in a dedicated alert script if the logic is too complex
